src/css/tabs/receiver.css

```diff
@@ -140,6 +140,36 @@
     background-color: #14407a;
 }
 
+/* from configuration tab */
```

Comment:
Is better to remove the comment or change from something more "general" like "rx configuration". Any future user will not know nothing about what was in the configuration tab and what not and can confuse them.